frontend/packages/dev-console/integration-tests/features/monitoring/monitoring-Page-Smoke-Metrics-ODC-4116.feature

```diff
@@ -0,0 +1,165 @@
+Feature: Metrics Tab on Monitoring Page
+    As a user, I should be able to run queries to check the usage
+
+
+Background: 
+    Given user is at Developer perspective
+
+
+@smoke, @regression
+Scenario: CPU Usage Query on Metrics tab: MH-01-TC04
+    Given user is on Monitoring page
+    When user clicks on Metrics tab
+    And user clicks on Select Query dropdown
+    And user selects CPU Usage Query
```

Comment:
Maybe you can use 
And user selects "CPU usage Query" from Select Query drop down

This will be easily understandable and number of lines gets reduced and also step is re-usable for other scenarios as well

Reply:
Updated